docs/add InstabugGRPCDataProtocol part in documentation (#5)
Instabug/Instabug-gRPC: README.md

1. Make sure you import our destination first: `import Instabug_gRPC_Swift`
2. Create Interceptor factory that confirms to the interceptor factory protocol that you have in your .grpc file
3. Make sure to return new instance of our Interceptor `InstabugClientInterceptor()` in the methods that you need us to log

> 4. Then conform on `InstabugGRPCDataProtocol` for request and response models which have data variable `{ get }` and convert your model to `Data` and return that `Data` in that variable for both request and response

Reviewer: What do you think?
```suggestion
4. Conform on `InstabugGRPCDataProtocol` for request and response models which requires that you expose your models as `Data`
```

1. Make sure you import our destination first: `import Instabug_gRPC_Swift`
2. Create Interceptor factory that confirms to the interceptor factory protocol that you have in your .grpc file
3. Make sure to return new instance of our Interceptor `InstabugClientInterceptor()` in the methods that you need us to log
4. Conform on `InstabugGRPCDataProtocol` for request and response models which requires that you expose your models as `Data`

> 5. You can convert your model to `Data` by conforming on `Encodable`

Reviewer: ```suggestion
5. You can convert your model to `Data` by conforming to `Encodable`
```

  }
}

extension GrpcAutomation_ServerErrorRequest: InstabugGRPCDataProtocol {
    public var gRPCRequestData: Data? {

>         return message.data(using: .utf8)

Reviewer: Can you make add an example with a more complex model that you convert to JSON then then return JSON data
